extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -221,40 +228,156 @@ class AmpAccordion extends AMP.BaseElement {
     const content = sectionComponents[1];
     const contentId = content.getAttribute('id');
     const isSectionClosedAfterClick = section.hasAttribute('expanded');
-    this.mutateElement(() => {
-      let toExpand;
-      if (opt_forceExpand === true) {
-        toExpand = true;
-      } else if (opt_forceExpand === false) {
-        toExpand = false;
-      } else {
-        toExpand = !section.hasAttribute('expanded');
-      }
+    let toExpand;
+    if (opt_forceExpand === true) {
+      toExpand = true;
+    } else if (opt_forceExpand === false) {
+      toExpand = false;
+    } else {
+      toExpand = !section.hasAttribute('expanded');
+    }
 
+    // Animate Toggle
+    if (this.element.hasAttribute('animate')) {
       if (toExpand) {
-        section.setAttribute('expanded', '');
         header.setAttribute('aria-expanded', 'true');
-        // if expand-single-section is set, only allow one <section> to be
-        // expanded at a time
+        this.animateExpand_(section).then(() => {
+        });
         if (this.element.hasAttribute('expand-single-section')) {
           this.sections_.forEach(sectionIter => {
             if (sectionIter != section) {
-              sectionIter.removeAttribute('expanded');
+              this.animateCollapse_(sectionIter);
```

Comment:
An open UX question here is whether we want to animate collapse when it's a side effect of expand on `expand-single-section`.

Reply:
Feedback is that yes we want to animate collapse.